grammar.js

```diff
@@ -254,17 +254,21 @@ module.exports = grammar({
 
         amount: $ => choice(
             seq(
-                $.quantity,
+                choice($.quantity, $.quantity_negative),
                 optional($.whitespace),
                 $.commodity),
             seq(
                 $.commodity,
                 optional($.whitespace),
-                $.quantity),
+                choice($.quantity, $.quantity_negative)),
         ),
 
         quantity: $ => seq(
-            optional('-'), /\d([\d., ]*\d)?/,
+            optional('+'), /\d([\d., ]*\d)?/,
```

Comment:
Can you extract `/\d([\d., ]*\d)?/` as `unsigned_quantity` or something? I'd like to avoid the proliferation of nasty regexes where possible :)

Also, while you're at it, do you mind renaming `quantity_negative` to `negative_quantity`?

Reply:
Excellent suggestions, thanks @cbarrete. This PR has been updated accordingly. I decided to name the extracted regex `_quantity`, but am happy to change it if you believe `unsigned_quantity` is a more suitable name.

Reply:
That's fine, thanks :)